utils/bots/TutorBot/cogs/TutorStaff.py

```diff
@@ -289,6 +291,8 @@ async def remove(self, ctx, id):
         description="Clear a tutoring session",
         guild_ids=[MAIN_ID.g_main, TUT_ID.g_tut]
     )
+    @permissions.has_any_role(MAIN_ID.r_tutor, guild_id=MAIN_ID.g_main)
+    @permissions.has_any_role(860932484206952468, guild_id=TUT_ID.g_tut)
```

Comment:
Same thing here.

Reply:
no, we can implement, this is just a short term fix bc the tutors cant use these cmds rn

Reply:
I think I found our issue lmao
![image](https://user-images.githubusercontent.com/56415283/153780680-6d8bbebd-9d2a-458d-9cd1-003f08385bcf.png)

Reply:
It doesn't even exist, it was probably returning the default ID we set which could have been outdated.

Reply:
we never used that ID in the first place, I added a new deco with a new ID since we moved from role name to ID, the "Tutor" role was only for main

Reply:
Ah alright, I guess we can stick with this until the PR to main.